src/App.vue

```diff
@@ -502,6 +504,23 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
       return
     }
 
+    // now that we have user info and org info, store the completing party
+    this.setCompletingParty({
+      firstName: this.getUserFirstName,
+      middleName: '',
+      lastName: this.getUserLastName,
+      mailingAddress: {
+        addressCity: this.getOrgInfo?.mailingAddress.city,
+        addressCountry: this.getOrgInfo?.mailingAddress.country,
+        addressRegion: this.getOrgInfo?.mailingAddress.region,
+        postalCode: this.getOrgInfo?.mailingAddress.postalCode,
+        streetAddress: this.getOrgInfo?.mailingAddress.street,
+        streetAddressAdditional: this.getOrgInfo?.mailingAddress.streetAdditional
+      },
+      email: this.getUserEmail,
+      phone: this.getUserPhone
```

Comment:
We might not use email or phone atm but we have the data so I thought I'd include it here. (It was hard to untangle how other UIs fetched this data so I made it clear here 😉 )